gp-ecommerce-fields/gpecf-deduct-deposit.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Gravity Perks // GP eCommerce Fields // Deduct Deposit from Order Summary
+ * https://gravitywiz.com/documentation/gravity-forms-ecommerce-fields/
+ *
```

Comment:
Let's add a brief description here and mention that there are changes that must be made inline.

Reply:
Done